stable/nfs-client-provisioner/templates/deployment.yaml

```diff
@@ -34,9 +34,6 @@ spec:
       affinity:
 {{ toYaml .Values.affinity | indent 8 }}
       {{- end }}
-      securityContext:
```

Comment:
I think I'd still include this, but have the default as `securityContext: {}` so that a user can define it if they want/can.

Reply:
Or you can allow user to customize their security context in the helm values.yml like mongodb do : 
- https://github.com/helm/charts/blob/master/stable/mongodb/templates/deployment-standalone.yaml#L64 
- https://github.com/helm/charts/blob/master/stable/mongodb/templates/deployment-standalone.yaml#L39

Reply:
Could this please be reverted first, later on we can do other fixes. It's breaking existing setups now.